server/index.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const db = require('../database/index');
+
+
+const app = express();
+app.use(bodyParser.json());
+app.use(morgan('dev'));
+
+let staticPath = __dirname + '/../public';
+console.log(staticPath);
+app.use('/books/:id', express.static(staticPath));
+
+// app.get('/books/:id', (req, res) => {
+//   const id = req.params.id;
+//   console.log(id);
+//   console.log('STATIC PAGE');
+//   res.send('hello get book id');
+// });
+
+//get initinal details
+app.get('/books/:id/details', (req, res) => {
+  const id = req.params.id;
+  db.getDetails(id)
+    .then(results => {
+      let details = results[0][0];
+      res.send(details);
+    })
+    .catch(err => {
+      console.log(err);
+      res.send(err);
+    });
+});
+
+//get data from either characters, awards, or editions table depending on table variable.
+app.get('/books/:id/details/:table', (req, res) => {
+  const id = req.params.id;
+  const table = req.params.table;
+  db.getRelatedData(table, id)
+    .then(results => {
+      let data = results[0];
+      res.send(data);
+    })
+    .catch(err => {
+      console.log('err getting extra details!', err);
+      res.send(err);
```

Comment:
you may also want to send the appropriate status code for error messages for a 200 status code will be parsed as success by the client

Reply:
Gotcha. Thanks for the great tip :)